conwayste/src/ui/ui_errors.rs

```diff
@@ -40,8 +40,11 @@ impl From<GameError> for Box<UIError> {
     }
 }
 
+/*
+ // This causes an infinite loop! FIXME: do we need this?
```

Comment:
Probably not. Much has to do be done with error handling (separately). Can just remove it for now?

Reply:
OK!